### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by dmgonch**
The changes fix existing build steps and add details for how to enable a parallel build.

### CODE REVIEW

**Comment by dmgonch** [comment]
@erikma: fyi

File: README.md [hunk 1]
```diff
@@ -4,12 +4,17 @@ acceleration benchmarking.
 # Quick start
 
     $ git clone https://github.com/benmcmorran/fleet.git
-    $ mkdir fleet/out
-    $ cmake -S fleet -B fleet/out -DTU_COUNT=10000 -DPROJECT_TYPE=flat
-    $ time cmake --build fleet/out
+    $ cd fleet
 
-This will create a library with 10,000 translation units (TUs) that have no
```
Comment (erikmav) [inline README.md:11]: Should keep explanatory text as the meaning of TU_COUNT is unclear otherwise

**Comment by dmgonch** [comment]
@erikma, @benmcmorran : please comment. thanks.

**Review by erikmav — APPROVED** [review]

**Comment by benmcmorran** [comment]
[Edited slightly](https://github.com/benmcmorran/fleet/pull/1/commits/3c2e87b50e0bf326e479a1b2cbba097cd207d32a) to explain the -j flag. Looks good to me.

File: README.md (hunk 1 above)
Comment (benmcmorran) [reply README.md:11]: Added back in.

**Pull request merged**